cmd/helm/status.go

```diff
@@ -124,6 +124,10 @@ func (s statusPrinter) WriteTable(out io.Writer) error {
 		fmt.Fprintf(out, "DESCRIPTION: %s\n", s.release.Info.Description)
 	}
 
+	if len(s.release.Info.Resources) > 0 {
+		fmt.Fprintf(out, "RESOURCES:\n%s\n", s.release.Info.Resources)
+	}
```

Comment:
The Helm [HIP on backwards compatibility](https://github.com/helm/community/blob/main/hips/hip-0004.md) covers CLI output and notes that structure can't change. Since Helm v3 has not showed the resources up to this point, displaying them needs to happen behind a flag. Could you please add a flag for this with a note to remove the flag in Helm v4.

Reply:
Updated implementation to move the new status functionality behind `--show-resources` flag as recommended